whois-api/src/main/java/net/ripe/db/whois/api/elasticsearch/ElasticFullTextIndex.java

```diff
@@ -33,7 +33,7 @@ public class ElasticFullTextIndex {
 
     @Autowired
     public ElasticFullTextIndex(final ElasticIndexService elasticIndexService,
-                                @Qualifier("jdbcSerialDao") final SerialDao serialDao,
+                                @Qualifier("jdbcSlaveSerialDao") final SerialDao serialDao,
```

Comment:
The `serialDao.getMaxSerialIdWithObjectCount()` method is annotated with a `REQUIRES_NEW` transaction, so may return a different serial than when using the `jdbcTemplate`. Maybe move that code into `JdbcRpslObjectOperations` so you can use the same `jdbcTemplate` instead ?

Reply:
I don't understand why is may return different serial.
We are just using jdbcTemplate once (JdbcSerialDao, line 61/62) so the `REQUIRES_NEW` should be fine

Reply:
A new transaction may give us a different view of the database, with different serials, if there have been any changes since the encompassing transaction was started

Reply:
we will ask Mahesh when he is back